hydrolib/core/dflowfm/tim/models.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+from typing import Callable, Dict, List
+
+from hydrolib.core.basemodel import ParsableFileModel, SerializerConfig
+
+from .parser import TimParser
+from .serializer import TimSerializer, TimSerializerConfig, TimTimeSerie
+
+
+class TimModel(ParsableFileModel):
+    """Model or tim data.
+
+    Attributes:
+        timeseries: List[TimTimeSerie]
+        serializer_config: TimSerializerConfig
+    """
+
+    serializer_config = TimSerializerConfig()
+    timeseries: List[TimTimeSerie]
```

Comment:
Was the following point addressed? 

> error bij meerdere keren dezelfde tijd

Reply:
This was discussed when the List was a dictionary, lets discuss further